apps/verticalhome/js/stores/item.js

```diff
@@ -217,8 +218,10 @@
           self.gridOrder = configurator.getGrid();
           self.populate(cb);
         });
-        window.addEventListener('configuration-pin-app-ready', function onPinAppReady() {
-          window.removeEventListener('configuration-pin-app-ready', onPinAppReady);
+        window.addEventListener('configuration-pin-app-ready',
```

Comment:
**conclusion**: using event name 'config-pa-ready' will give us 78 symbols in line 221 and 72 in the line 222 - PROFIT!!!